js_modules/dagit/packages/core/src/runs/RunCreatedByCell.tsx

```diff
@@ -19,15 +19,21 @@ export function RunCreatedByCell(props: Props) {
   const scheduleTag = tags.find((tag) => tag.key === DagsterTag.ScheduleName);
   const sensorTag = tags.find((tag) => tag.key === DagsterTag.SensorName);
   const user = tags.find((tag) => tag.key === DagsterTag.User);
-
-  const jsx = [];
+  const automaterialize = tags.find(
+    (tag) =>
+      tag.key === DagsterTag.Automaterialize ||
+      // Backwards compatibility
+      (tag.key === 'created_by' && tag.value === 'auto_materialize'),
+  );
+  const createdBy = tags.find((tag) => tag.key === 'created_by');
```

Comment:
secret feature, you can set `created_by` tag and it'll show up in the runs table